src/test/java/uk/gov/justice/hmpps/prison/util/builders/OffenderBookingBuilder.kt

```diff
@@ -82,6 +89,33 @@ class OffenderBookingBuilder(
       .exchange()
       .expectStatus().isOk
       .returnResult<InmateDetail>().responseBody.blockFirst()!!.also {
+      if (released) {
```

Comment:
convent method of creating an inactive booking; book them in and then release them